pymobiledevice3/services/mobilebackup2.py

```diff
@@ -252,7 +256,7 @@ def version_exchange(self, dl: DeviceLink, local_versions=None) -> None:
         assert reply[0] == 'DLMessageProcessMessage' and reply[1]['ErrorCode'] == 0
         assert reply[1]['ProtocolVersion'] in local_versions
 
-    def init_mobile_backup_factory_info(self, afc: AfcService):
+    def init_mobile_backup_factory_info(self, afc: AfcService) -> Mapping[str, PlistTypes]:
```

Comment:
Use Any instead of PlistTypes (or just mapping)

Reply:
1